core/server/apps/subscribers/lib/router.js

```diff
@@ -79,18 +76,18 @@ function handleSource(req, res, next) {
 function storeSubscriber(req, res, next) {
     req.body.status = 'subscribed';
 
-    if (_.isEmpty(req.body.email)) {
+    if (req.body.email.length === 0) {
```

Comment:
I am not sure about this change. This is a public post endpoint. You won't bring the server down, but you would still be able to throw a type error.

Reply:
@kirrg001 Since _ was removed then I used this check. I don't know if it's necessary to use this method instead of a `length` check.